torch/csrc/jit/api/module_save.cpp

```diff
@@ -5,35 +5,45 @@ namespace torch {
 namespace jit {
 
 void Module::save(std::ostream& out, const ExtraFilesMap& extra_files) const {
-  ExportModule(*this, out, extra_files, false /* bytecode_format */);
+  ExportModule(
+      *this,
+      out,
+      extra_files,
+      at::optional<uint64_t>() /* bytecode_format version*/);
```

Comment:
If not provided then we use the current (aka kProducedBytecodeVersion) right?

Reply:
This api is used for saving jit model. For the api `torch.jit.save` to save jit model, we don't generate bytecode.

Reply:
I think if not provided it will not dump the bytecode part. Not sure if it's the perfect API, but comments should be added to explain the meaning of this argument.